app/views/calculators/leave_pot_untouched/_estimate.html.erb

```diff
@@ -21,11 +21,14 @@
 
 <ul class="t-calculator-notes">
   <li>
-    This is an estimate based on your pot growing at a rate of about
+    This is an estimate based on your whole pot growing at a rate of about
     <%= number_to_percentage(LeavePotUntouchedCalculator::INTEREST_RATE * 100) %>
     per year — this may vary.
   </li>
   <li>
     The amount in your pot will be affected by inflation and any fees your provider charges.
   </li>
+  <li>
+  You must leave your whole pot – if you take a 25% tax-free lump sum, you must take the rest of your money within 6 months, using one of the [other pension options](/pension-pot-options)
```

Comment:
Perfect. I've squashed your changes together into a single edit so that the history of this file is easy to understand. I'll merge for you now!